girder/test/test_size.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2013 Kitware Inc.
```

Comment:
What is our policy on updating this dates?

Reply:
No idea... I'd love to remove this header from all the files

Reply:
I thought we decided the date should be in just one file and we could remove it elsewhere.

Reply:
Okay, we can take care of that elsewhere.